jenkins/README.md

```diff
@@ -33,28 +33,13 @@ The job result will be posted as a comment.
 * **/test-v1a4-centos-integration** run integration tests for v1alpha4 on
    CentOS
 
-It is also possible to prevent any job run by adding **/skip-test** in the PR
-description.
-
-If the author is not in the whitelist but should be trusted then by adding a
-comment **add to whitelist** on the PR, the author will then be able to run the
-jobs on its own.
-
 ## Cloud Resources cleanup
 
 There is a Jenkins [master job](https://jenkins.nordix.org/view/Airship/job/airship_master_integration_tests_cleanup/)
 that cleans up all the leftover VMs from
 [CityCloud](https://www.citycloud.com/) every 6 hours which has failed to be
 deleted at the end of v1alphaX integration test.
 
-## "Can one of the admins verify this patch?"
-
-For all the PRs from authors that are not whitelisted, the bot will add a
-comment "*Can one of the admins verify this patch?*". This means that the author
-is not in the whitelist and that someone from the metal3-io organization should
-review the PR and run the tests (with */test-integration*) or add the author to
-whitelist if trusted.
-
```

Comment:
instead of removing, it can be updated to

```
. Thanks for your PR.

I'm waiting for a metal3-io member to verify that this patch is reasonable to test. If it is, they should reply with /ok-to-test on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should join the org to skip this step.

Once the patch is verified, the new status will be reflected by the ok-to-test label.

I understand the commands that are listed here.
```
example: https://github.com/metal3-io/baremetal-operator/pull/964#issuecomment-911958716

Reply:
Updated